test/smoke/appServers/Jetty.9/jre.excludes.txt

```diff
@@ -0,0 +1,2 @@
+openjdk:7-jre
+azul/zulu-openjdk:7
```

Comment:
I believe you mentioned you were planning to remove zulu-jdk. Has that changed or are you addressing it in separate PR?

Reply:
I'm going to add that to a different one: #642